taxonomy/migrations/0025_industry.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.15 on 2022-10-17 06:41
+
+from django.db import migrations, models
+from taxonomy.constants import NAICS2_CODES
+
+
+def add_industry_data(apps, schema_editor):
+    """
+    Add industry data from NAICS2 codes added in the constants.
+    """
+    Industry = apps.get_model('taxonomy', 'Industry')
+    for code, industry_name in NAICS2_CODES.items():
+        Industry.objects.create(code=code, name=industry_name)
+
+
+def do_nothing(apps, schema_editor):
+    """
+    Do nothing.
+    """
```

Comment:
should we use the keyword `pass` here?

Reply:
no need for `pass`, empty docstring is enough for empty function or empty class.